syntax = "proto3";

package yandex.cloud.priv.iam.v1;

import "cloud/identity/proto/iam/v1/user_account.proto";

option go_package = "iam";
option java_outer_classname = "PUAS";

service UserAccountService {
  rpc Get (GetUserAccountRequest) returns (UserAccount) {
  }
}

message GetUserAccountRequest {
  string user_account_id = 1;
}
